common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -289,6 +314,7 @@ def test_country_field(self):
             u'',
             [u'Pakistan', u''],
         )
+        self.assert_event_emitted_num_times('country', 2)
```

Comment:
This line seems to be failing on jenkins again. Locally we get 2 country events, but 4 on jenkins.